_docs/tasks/rate-limiting.md

```diff
@@ -44,28 +47,79 @@ Using Istio we can ensure that `1qps` is not breached.
    If you log in as any other user (or logout) you should see red ratings stars with each review,
    indicating that the `ratings` service is being called by the "v3" version of the `reviews` service.
 
-1. Configure mixer with the rate limit. 
-
-   Save this as ratelimit.yaml: 
+1. Configure memquota adapter with rate limits. 
+   
+   Save the following file as ratelimit-handler.yaml.
 
    ```yaml
-   rules:
-   - aspects:
-     - kind: quotas
-       params:
-         quotas:
-         - descriptorName: RequestCount
-           maxAmount: 1
-           expiration: 1s
+   apiVersion: config.istio.io/v1alpha2
+   kind: memquota
+   metadata:
+     name: handler
+     namespace: istio-config-default
+   spec:
+     quotas:
+     - name: requestcount.quota.istio-config-default
+       # default rate limit is 5000qps
+       maxAmount: 5000
+       validDuration: 1s
+       # The first matching override is applied.
+       # A requestcount instance is checked against override dimensions.
+       overrides:
+       # The following override applies to traffic from 'rewiews' version v3,
+       # destined for the ratings service. The destinationVersion dimension is ignored.
+       - dimensions:
+           destination: ratings
+           source: reviews
+           sourceVersion: v3
+         maxAmount: 1
+         validDuration: 1s
    ```
 
    and then run the following command:
 
    ```bash
-   istioctl mixer rule create global ratings.default.svc.cluster.local -f ratelimit.yaml
+   istioctl create -f ratelimit-handler.yaml
+   ```
+   
+   If you log in as "jason", you will be subject to the default `5000 qps` rate limit.
+   Since all others are routed via reviews-v3, they are limited to `1 qps`.
```

Comment:
This wording is confusing. Sounds like user should log in as "jason" now, even though we haven't yet activated the quota. I would suggest changing it to something like this:

This handler specifies that only calls to the ratings service via reviews-v2 (user "jason" in our example)
will be subject to the default `5000 qps` rate limit.
All other users (since they are being routed via reviews-v3) will be limited to `1 qps`.